openedx_learning/__init__.py

```diff
@@ -1,4 +1,4 @@
 """
 Open edX Learning ("Learning Core").
 """
-__version__ = "0.2.5"
+__version__ = "0.2.6"
```

Comment:
Personally I wouldn't bother including a version bump for a small change like this, unless it's blocking some other PR like an edx-platform PR.

Reply:
Agreed! Removed the version bump.